projects/envoy/#build.sh#

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash -eu
```

Comment:
looks like this was added accidentally :)

Reply:
Aaaah!!! Thank you.